spec/jobs/webflow_sync/create_item_job_spec.rb

```diff
@@ -91,5 +98,24 @@ module WebflowSync
         expect(article.reload.webflow_slug).to be_nil
       end
     end
+
+    context 'when publish_to_all_domains is true' do
+      before(:each) do
+        @publish_to_all_domains = WebflowSync.configuration.publish_to_all_domains
+        WebflowSync.configure do |config|
+          config.publish_to_all_domains = true
+        end
+      end
+
+      after(:each) do
+        WebflowSync.configure do |config|
+          config.publish_to_all_domains = @publish_to_all_domains
+        end
+      end
+
+      it 'publishes all domains', vcr: { cassette_name: 'webflow/create_item_and_publish', record: :once } do
```

Comment:
Remove `record: :once` here

Reply:
How do we know this publishes all domains?

We could set the test here to check if `publish` method was called or not.